apps/omg_status/lib/omg_status/metric/vmstats_sink.ex

```diff
@@ -29,9 +29,38 @@ defmodule OMG.Status.Metric.VmstatsSink do
 
   defp base_key, do: Application.get_env(:vmstats, :base_key)
 
-  def collect(:counter, key, value), do: _ = Datadog.set(key, value)
+  def collect(:counter, key, value), do: _ = Datadog.set(key, value, tags: ["application:#{get_application_mode()}"])
 
-  def collect(:gauge, key, value), do: _ = Datadog.gauge(key, value)
+  def collect(:gauge, key, value), do: _ = Datadog.gauge(key, value, tags: ["application:#{get_application_mode()}"])
 
-  def collect(:timing, key, value), do: _ = Datadog.timing(key, value)
+  def collect(:timing, key, value), do: _ = Datadog.timing(key, value, tags: ["application:#{get_application_mode()}"])
```

Comment:
I'm probably missing context, but why couldn't we just pick this value up from the environment? We already specify a value here at `ELIXIR_SERVICE` which returns `CHILDCHAIN` or `WATCHER`.

Reply:
Because the elixir service environment variable is deprecated (makes no sense) if you’re running a release. Which we don’t yet. But it’s easier if I don’t rely on it.